python/tests/test_benchmark.py

```diff
@@ -48,6 +48,11 @@ def assert_knn_equal(
     # set total_tol because two nearest neighbors may have the same distance to the query
     assert array_equal(mg_indices, indices, total_tol=total_tol)
 
+    for i in range(len(mg_indices)):
```

Comment:
Sorry I missed the previous assert on distances.  I agree that this check I requested is redundant in that case.